readthedocs/settings/base.py

```diff
@@ -63,6 +63,15 @@ def _show_debug_toolbar(request):
 
         return {
             "SHOW_TOOLBAR_CALLBACK": _show_debug_toolbar,
+            "DISABLE_PANELS": [
+                # Default ones
+                "debug_toolbar.panels.profiling.ProfilingPanel",
+                "debug_toolbar.panels.redirects.RedirectsPanel",
+                # Custome ones
+                # We are disabling these because they take a lot of time to execute in the new dashboard
```

Comment:
If this is a problem with the new dashboard only, it probably points to a problem there.

Reply:
Yeah, we make an intensive usage of the `include` template tag. It's a "known issue/bug" --there is no solution as far as we can tell. @agjohnson and myself debugged this _a lot_ in the past.

Reply:
probably worth noting that in the comment